FBTweak/FBTweakCategory.h

```diff
@@ -11,30 +11,49 @@
 
 @class FBTweakCollection;
 
+@protocol FBTweakCategory <NSObject>
+
 /**
-  @abstract A named grouping of collections.
+ @abstract The name of the category.
  */
-@interface FBTweakCategory : NSObject <NSCoding>
+@property (nonatomic, copy, readonly) NSString *name;
 
 /**
-  @abstract Creates a tweak category.
-  @discussion This is the designated initializer.
+ @abstract The collections contained in this category.
  */
-- (instancetype)initWithName:(NSString *)name;
+@property (nonatomic, copy, readonly) NSArray<FBTweakCollection *> *tweakCollections;
 
 /**
-  @abstract The name of the category.
+ @abstract Resets all tweaks in \c tweakCollection to their default value.
  */
-@property (nonatomic, copy, readonly) NSString *name;
+- (void)reset;
+
+@optional
+
+/**
+ @abstract Updates \c tweakCollections to the latest value, and calls \c completion when done.
+ If there was an error while updaing, \c error contains the error, and if the update was successful,
+ \c error is \c nil.
+ @param completion Completion block to be called when the update is complete.
```

Comment:
Please specify whether the \c completion block is executed synchronously or not.

Reply:
Changed the text